### PR TITLE
NEW Use Config for determining Vary header

### PR DESCRIPTION
**Pull request by nyeholt**
Existing implementation hardcodes the Vary header; swap to using Config layer
instead

Added test for changing the variable from config

### CODE REVIEW

**Comment by nathanbrauer** [comment]
:+1:

**Comment by nathanbrauer** [comment]
fyi @tractorcow

File: control/HTTP.php [hunk 1]
```diff
@@ -358,7 +358,10 @@ public static function add_cache_headers($body = null) {
 
 			// To do: User-Agent should only be added in situations where you *are* actually
 			// varying according to user-agent.
-			$responseHeaders['Vary'] = 'Cookie, X-Forwarded-Protocol, User-Agent, Accept';
+            $vary = $config->get('HTTP', 'vary');
+            if ($vary && strlen($vary)) {
+                $responseHeaders['Vary'] = $vary;
```
Comment (tractorcow) [inline control/HTTP.php]: Mixed tabs and spaces? (yes I have magic eyes)

Comment (nyeholt) [inline control/HTTP.php]: Urg, any idea on how to run Netbeans with per-project formatting specifics? :D

Comment (stevie-mayhew) [inline control/HTTP.php]: @nyeholt https://github.com/welovecoding/editorconfig-netbeans :smile:

Comment (nyeholt) [inline control/HTTP.php]: Thanks @stevie-mayhew !

**Comment by nyeholt** [comment]
Have added another commit that uses tabs :) I was going to squash + forcepush, but figure it might be useful having the master-compatible commit as a separate thing?

**Pull request merged**